chromium_src/tools/grit/preprocess_if_expr.py

```diff
@@ -43,6 +43,14 @@ def maybe_keep_upstream_version(override_in_folder, out_folder, override_file):
             return overridden_name
     return None
 
+def run_mangler(mangler_file, preprocess_file):
+    """Runs the mangler on the given file"""
+    import subprocess
+
+    # Note: We read from and write to the preprocess file - this way any
+    # preprocessing that upstream does will be mangled.
+    subprocess.run(['npx', 'tsx', '--tsconfig', '../../brave/tsconfig.json', '../../brave/build/commands/scripts/lit_mangler.ts', 'mangle', '-m', mangler_file, '-i', preprocess_file, '-o', preprocess_file])
```

Comment:
Hey @thypon this seems like it'd be a good candidate for a `semgrep` rule - you shouldn't be able to execute `npx` without the `--no` flag